.eslintrc.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
       extends: ['plugin:@typescript-eslint/eslint-recommended'],
       files: ['*.ts', '*.tsx'],
       parser: '@typescript-eslint/parser',
-      plugins: ['@typescript-eslint/eslint-plugin'],
+      plugins: ['@typescript-eslint/eslint-plugin', 'local'],
       rules: {
         '@typescript-eslint/array-type': ['error', {default: 'generic'}],
         '@typescript-eslint/ban-types': [
```

Comment:
😱 almost forgot the whole point of this PR: to set this back to just `error` so that the rest of the codebase is checked normally